### DISCUSSION TITLE
Pan and zoom only one Y axis

### DISCUSSION THREAD

**Discussion by Tesla2k** [chartjs/chartjs-plugin-zoom#539] [Q&A]
Hello,
I have a line chart, with pan and zoom enabled.
I have several datasets on this chart. 
Now I want one dataset not to be changed by pan or zoom in the Y axis, the x axis should behave normal. The goal is, that this line is always visible independent where you pan or zoom. My first idea was to put it on a different Y axis, but I didn't find a way not to pan this axis.

(This dataset is a status line filled with 1 when the datapoints a valid and null when not. )

**Comment by kurkle** [comment]
After #524 (released in version 1.1.0), you should be able to use the limits to freeze an axis.

For example: (the frozen axis id is assumed to be `y2`)

```js
const config = {
  // ...
  options: {
    plugins: {
      zoom: {
        limits: {
          y2: {
            min: 0,
            max: 1,
            minRange: 1
          }
        }
      }
    }
  }
};
```
